samples/interactive-tutorials/src/test/java/product/setup/ProductsCreateBigqueryTableTest.java

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
The setup scripts are only used one-time for DPE team to initialize the test data, and kept there in case we'll need to recreate the test data. We should not add tests for them (which will unexpectedly tear down/recreate the existing test data).

Reply:
Removed tests for setup/cleanup.